operator/pkg/helmreconciler/prune.go

```diff
@@ -147,6 +148,14 @@ func (h *HelmReconciler) DeleteObjectsList(objectsList []*unstructured.Unstructu
 					h.opts.Log.LogAndPrintf("object: %s is not being deleted because it no longer exists",
 						obj.Hash())
 				}
+			} else {
+				objGvk := o.GroupVersionKind()
+				gvkStr := fmt.Sprintf("%s/%s/%s", objGvk.Group, objGvk.Version, objGvk.Kind)
```

Comment:
i think there's a utility function somewhere for getting a gvk string already, let's use that instead. otherwise, let create a helper in object pkg.

Reply:
That is removed.